### PR TITLE
added instructions on how to use merge

### PR DESCRIPTION
**Pull request by nicflores**
Added some instructions with the hope it helps some of the new folks.

### CODE REVIEW

File: README.md
Comment (sellout) [inline README.md]: Is this actually enforced by the script and/or GitHub? That’s great if so. If not, I’d take a RFC-like approach and say “should” instead of “must”.

File: README.md
Comment (drostron) [inline README.md]: s/exchanged/exchange/

File: README.md
Comment (drostron) [inline README.md]: Maybe "Merge Instructions" since devtools is for more than merge.

File: README.md
Comment (drostron) [inline README.md]: This might be better suited for a general merge process document. Also, it's not required for the merge script to work and is per PR rather than initial setup.

File: README.md
Comment (drostron) [inline README.md]: s/wihtin/within/

Being within the `devtools` directory isn't required.

**Review by drostron — CHANGES_REQUESTED** [review]
Thanks for the README update! Just a few comments.

If you assign me to the PR I'll merge once comments are addressed.

**Comment by drostron** [comment]
Also looks like the CLA needs signing.

File: README.md
Comment (sellout) [inline README.md]: Somewhere around here, you also want to ensure the PR has a “version:” label on it.

**Review by sellout — CHANGES_REQUESTED** [review]
Ideally, this documentation would be on the `merge` script itself rather than more distant from the implementation. There’s already some there, but it could be fleshed out more like this is. A `usage()` function (or even a `man` page) would also be useful. Then the README could simply say “Run `merge -h` for instructions.”

@mbbx6spp has a bunch of great style guides, and this is part of the bash one: https://github.com/mbbx6spp/styleguides/tree/master/bash#usage

**Comment by drostron** [comment]
👍🏾 for the `merge -h` suggestion.

File: README.md
Comment (nicflores) [reply README.md]: fixed wording

File: README.md
Comment (nicflores) [reply README.md]: added suggested title

File: README.md
Comment (nicflores) [reply README.md]: I went ahead and removed step 0.

File: README.md
Comment (nicflores) [reply README.md]: updated wording

File: README.md
Comment (nicflores) [reply README.md]: This isn't enforced by the script, and I'm not sure about GitHub enforcing it but I'll test this next time I do a PR. I went ahead and removed this step from the instructions. Once I know how GitHub behaves I'll update the usage.

File: README.md
Comment (nicflores) [reply README.md]: added comment about adding version to PR

**Comment by nicflores** [comment]
Added usage function to merge script along with command line arguments.

File: bin/merge
Comment (drostron) [inline bin/merge]: Thoughts on the change from the current interface? A repo can never be `-h` right? The following might be reasonable.
`Usage: $0 [-h] <REPO-NAME> <PR-NUMBER>`

File: bin/merge
Comment (drostron) [inline bin/merge]: s/merge/Merge/

File: bin/merge
Comment (drostron) [inline bin/merge]: Should this be `-r <REPO-NAME> <PR-NUMBER>`?

File: bin/merge
Comment (drostron) [inline bin/merge]: I believe you're looking for separate params here?

File: bin/merge
Comment (drostron) [inline bin/merge]: Same

File: README.md
Comment (drostron) [inline README.md]: Update might be required depending on the answer to the interface question below.

File: bin/merge
Comment (drostron) [inline bin/merge]: Perhaps stale depending on whether `-r` sticks around.

**Review by drostron — CHANGES_REQUESTED** [review]
Nice to see merge gaining `-h`.

File: bin/merge
Comment (nicflores) [reply bin/merge]: Since I'm using `getopts` to parse the flags and arguments, `<REPO-NAME> <PR-NUMBER>` need to come with a flag.

What about this?
`-r <REPO-NAME>,<PR-NUMBER>`

File: bin/merge
Comment (nicflores) [reply bin/merge]: fixed

File: bin/merge
Comment (nicflores) [reply bin/merge]: see my comment above

File: bin/merge
Comment (nicflores) [reply bin/merge]: same reason as above.

File: bin/merge
Comment (sellout) [reply bin/merge]: You can have non-flagged arguments with getopts, you just have to handle them after the getopts block.

Comment (nicflores) [reply bin/merge]: With the latest push:

The following two runs of `merge` will print out its usage.
`./merge`
`./merge -h`

Once you need to merge you can run it as usual:

`./merge quasar-analytics/quasar 2030`

File: bin/merge
Comment (nicflores) [reply bin/merge]: I've cleaned this up.

**Review by drostron — APPROVED** [review]

**Comment by drostron** [comment]
LGTM, will merge once @sellout signs off.

File: README.md
Comment (sellout) [inline README.md]: This still has the `-r` flag.

**Review by sellout — CHANGES_REQUESTED** [review]
Just the one correction to the README, and I’m 👍🏾

File: README.md
Comment (nicflores) [reply README.md]: good catch!
fixed and pushed.

**Review by sellout — APPROVED** [review]

**Pull request merged**
